apps/meteor/app/api/server/v1/voip/rooms.ts

```diff
@@ -89,10 +90,11 @@ API.v1.addRoute(
 				token: String,
 				agentId: Match.Maybe(String),
 				rid: Match.Maybe(String),
+				direction: VoipRoomDirectionValidator,
```

Comment:
What if instead of having it as optional, you move the check to inside the `!rid` if? 👀 That way you don't have to check for the prop to not to be optional